docs/specs/schema.yml

```diff
@@ -55,3 +55,40 @@ outputs:
       "content": { "title": "my title" }
     }
   build.manifest:
+---
+# Support schema documents with Href content
+inputs:
+  docfx.yml:
+  # TODO: why is there no output for this toc file?
+  TOC.yml:
+  docs/TOC.yml:
+  docs/a.yml: |
+    #YamlMime:ContextObject
+    brand: azure
+    breadcrumb_path: TOC.yml
+    toc_rel: ../TOC.yml
+outputs:
+  docs/TOC.json:
+  docs/a.json: |
+    {
+      "content": { "brand": "azure", "breadcrumb_path": "TOC.json", "toc_rel": "../TOC.json" },
+    }
+  build.manifest:
+---
+# schema documents with Href content but wrong type
+inputs:
+  docfx.yml:
+  # TODO: why is there no output for this toc file?
+  TOC.yml:
+  docs/TOC.yml:
+  docs/a.yml: |
+    #YamlMime:ContextObject
+    brand: azure
+    breadcrumb_path: 1
+    toc_rel: ../TOC.yml
+outputs:
+  docs/TOC.json:
+  build.manifest:
+  build.log: |
+   ["error","violate-schema","(Line: 3, Character: 18) Field with attribute 'Microsoft.Docs.Build.HrefAttribute' should be of type System.String.","docs/a.yml",3,18]
```

Comment:
if error has structured line/column, do we still need them in error messages?

Reply:
we had them in other messages as well, won't hurt in both?

---
In reply to: [210853918](https://github.com/dotnet/docfx/pull/3209#discussion_r210853918) [](ancestors = 210853918)

Reply:
Will create a new PR to remove line info from error messages

---
In reply to: [210855794](https://github.com/dotnet/docfx/pull/3209#discussion_r210855794) [](ancestors = 210855794,210853918)